src/wallet/wallet.cpp

```diff
@@ -2311,18 +2311,24 @@ DBErrors CWallet::LoadWallet()
     return nLoadWalletRet;
 }
 
-DBErrors CWallet::ZapSelectTx(std::vector<uint256>& vHashIn)
+void CWallet::ClearTxns(const std::vector<uint256>& tx_hashes)
```

Comment:
In 559529480fedd4181dab76f4452b5998d0cdadc3 "refactor: decouple wallet EraseTxns from ZapSelectTx"

The commit message calls this `EraseTxns`.

Reply:
fixed